ios/Exponent/Versioned/Modules/Api/EXNotifications.m

```diff
@@ -40,6 +41,18 @@ - (void)setBridge:(RCTBridge *)bridge
   _experienceId = _bridge.experienceScope.experienceId;
 }
 
+
+RCT_REMAP_METHOD(getDevicePushTokenAsync,
+                 getDevicePushTokenAsyncWithResolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+  NSData *token = [[NSUserDefaults standardUserDefaults] apnsToken];
+  if (!token) {
+    return reject(0, @"APNS token has not been set", nil);
+  }
+  return resolve([token apnsTokenString]);
```

Comment:
@terribleben Should this formatting be in the native code or in expo-sdk: https://github.com/expo/expo-sdk/pull/64/files#diff-af27b193cae3940470461d4af8fa95daR135 ?